### DISCUSSION TITLE
Anyone else not able to add version notes when saving?

### DISCUSSION THREAD

**Discussion by passioneight** [pimcore/pimcore#12558] [Q&A / Support] [answered]
Using Pimcore X, it seems code like the following does not add version notes anymore:

```php
$element->save(['versionNote' => 'Test version note']);
```

> According to the [docs](https://pimcore.com/docs/pimcore/current/Development_Documentation/Documents/Working_with_PHP_API.html#page_CRUD-Operations) and to some [comments in the code](https://github.com/pimcore/pimcore/blob/58b7dfc2950db427f06159bb9e5dc2d3b9a2ab26/models/Document.php#L355) this should work.

Can someone confirm this? The related code is rather confusing, so I'm not entirely sure whether I'm overlooking something or this is a bug.

**Comment by kingjia90** [comment, marked as answer]
It seems a bug only affecting Documents, i've proposed a potential fix here https://github.com/pimcore/pimcore/pull/12652 will be available in 10.4.2 and 10.5
